addons/html_builder/static/src/plugins/button_option.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.ButtonOption">
+    <BuilderRow label.translate="Style" level="1">
```

Comment:
Where did you find this code ? (The legacy version)

Reply:
We should ask to lebl if we want to keep it or where we want to move it because we don t have the toolbar in the sidebar Builder anymore